civictechprojects/static/css/ProjectSearchBar.css

```diff
@@ -32,4 +32,4 @@
 button.ProjectSearchBar-submit:disabled {
   background-color: #ededed;
   color: #7a7a7a;
-}
\ No newline at end of file
+}
```

Comment:
Any changes in this file?

Reply:
No, I had some .css there before I moved the button into the TagContainer instead of the search bar. There shouldn't be any changes to this file at all. Maybe Atom likes adding/removing newlines?